homeronet/Plugin/SBEmail.cs

```diff
@@ -0,0 +1,32 @@
+﻿using homeronet.Messages;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace homeronet.Plugin {
+
+    public class SBEmail : IPlugin {
+        private List<string> _registeredCommands = new List<string>() { "sbemail" };
+
+        public void Startup() {
+        }
+
+        public void Shutdown() {
+        }
+
+        public Task<IStandardMessage> ProcessTextCommand(ITextCommand command) {
+            return new Task<IStandardMessage>(() => {
+                var rnd = new Random().Next(206);
```

Comment:
Gotta wonder if we should move that to a const. Who am I kidding, sbemail will never update. :(